jupyter-scipy/rockcraft.yaml

```diff
@@ -0,0 +1,175 @@
+name: jupyter-scipy
+summary: An image for using Jupyter SciPy
+description: |
+  This image is used as part of the Charmed Kubeflow product. It is deployed
+  in a user namespace, and enables users to perform CPU-accelerated tasks
+  using SciPy from within a Jupyter Notebook.
+
+  Both SciPy and Jupyter are installed in Conda environment, which is
+  automatically activated.
+version: v1.7.0_22.04_1 # version format: <KF-upstream-version>_<base-version>_<Charmed-KF-version>
+license: Apache-2.0
+base: ubuntu:22.04
+services:
+  jupyter:
+    override: replace
+    summary: "jupyter-scipy service"
+    startup: enabled
+    user: jovyan
+    environment:
+     NB_USER: jovyan
+     NB_UID: 1001
+     NB_PREFIX: /
+     HOME: /home/jovyan
+     SHELL: /bin/bash
+     LANG: en_US.UTF-8
+     LANGUAGE: en_US.UTF-8
+     LC_ALL: en_US.UTF-8
+     PATH: /opt/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+    command: /opt/conda/bin/jupyter lab --notebook-dir="/home/jovyan" --ip=0.0.0.0 --no-browser --port=8888 --ServerApp.token="" --ServerApp.password="" --ServerApp.allow_origin="*" --ServerApp.base_url="/" --ServerApp.authenticate_prometheus=False
+platforms:
+  amd64:
+
+parts:
+  overlay-pkgs:
```

Comment:
Is this part actually doing something ?

Reply:
Looks like these packages should be made available to other parts:
https://canonical-craft-parts.readthedocs-hosted.com/en/latest/explanation/overlays.html#overlay-package-installation